contributors.yml

```diff
@@ -487,7 +492,3 @@
 - zainfathoni
 - zayenz
 - zhe
-- mitchelldirt
```

Comment:
I moved these contributors to make the list be in alphabetical order since it seems like it was that way at some point- not sure if that matters, I can remove this change if we don't need it

Reply:
Please revert those changes, the list will get alphabetically sorted by our CI, and you may be generating conflicts here.

Just add your username in the right alphabetical spot, or at the bottom of the list.

Reply:
@machour I reverted it and added my username into the list!